packages/amplify-migration-codegen-e2e/src/__tests__/migration_codegen_e2e.test.ts

```diff
@@ -21,11 +26,16 @@ void describe('Migration Codegen E2E tests', () => {
     await cleanupProjects(projRoot);
   });
 
-  void it('performs full migration codegen flow with Auth backend', async () => {
+  void it('performs full migration codegen flow with backend', async () => {
     await setupAndPushGen1Project(projRoot, 'CodegenTest');
-    const { gen1UserPoolId, gen1Region } = await assertGen1Setup(projRoot);
+    const { gen1UserPoolId, gen1FunctionName, gen1BucketName, gen1GraphQLAPIId, gen1Region } = await assertGen1Setup(projRoot);
```

Comment:
gen1GraphQLAPIId -> gen1GraphqlApiId
gen1Region -> should it be a static variable instead of passing back arg? How the region is set up in the existing gen1 integ test?